frontier/core/src/main/java/com/zhaw/frontier/entities/Walls.java

```diff
@@ -0,0 +1,53 @@
+package com.zhaw.frontier.entities;
+
+import com.badlogic.ashley.core.Component;
+import com.badlogic.ashley.core.Engine;
+import com.badlogic.ashley.core.Entity;
+import com.badlogic.gdx.graphics.Pixmap;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.zhaw.frontier.components.BuildingPositionComponent;
+import com.zhaw.frontier.components.HealthComponent;
+import com.zhaw.frontier.components.RenderComponent;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ */
+public class Walls {
+
+    /**
+     *
+     * @param engine
+     * @return
+     */
+    public static Entity createDefaultWall(Engine engine) {
+        List<Component> components = new ArrayList<>();
+        components.add(new BuildingPositionComponent());
+        components.add(new HealthComponent());
+
+        RenderComponent renderComponent = new RenderComponent();
+
+        //TODO change placeholder for the actual wall texture
+        Texture texture = createPlaceHolder();
+        renderComponent.sprite = new Sprite(texture);
+
+        components.add(renderComponent);
+        return EntityFactory.buildEntity(engine, components);
```

Comment:
```suggestion
        var entity = new Entity();
        entity.add(new BuildingPositionComponent());
        entity.add(new HealthComponent());

        RenderComponent renderComponent = new RenderComponent();

        //TODO change placeholder for the actual wall texture
        Texture texture = createPlaceHolder();
        renderComponent.sprite = new Sprite(texture);

        entity.add(renderComponent);
        return entity;
```
engine.addEntity(entity);
must tbe done outside of the method.